src/Propel/Runtime/ActiveQuery/Criteria.php

```diff
@@ -2295,38 +2102,6 @@ public function quoteIdentifier($string, $tableName = '')
         return $string;
     }
 
-    /**
-     * @param string $string
-     *
-     * @return string
-     */
-    public function quoteIdentifierTable($string)
```

Comment:
I would consider this as a BC break.
This is a internal class, but at least for us Criteria class is widely used.
@dereuromark I haven't found usage of it in our code base, but still.

Just to be sure, this is part of the major release, right?

Reply:
Yes, a beta release may do small BC breaks where justified. We outline those in the release notes.